site/assets/js/application.js

```diff
@@ -101,6 +101,16 @@
     })
   }
 
+  var rootElementExample = document.getElementById('modal-root-example')
```

Comment:
you can avoid all this change using the `data-bs-root-element` markup on docs

Reply:
Can you give an example? I can't see `data-bs-root-element` anywhere in code to see what it does

1. Is it something you suggest I implement?
2. Or is it implemented already like below?
```
<div id="modal-root-example" class="modal fade" data-bs-root-element="#another-element" ... >
```

Reply:
Number 2. 

```js 


  _getConfig(config) {
    config = {
      ...Default,
      ...Manipulator.getDataAttributes(this._element),
      ...(typeof config === 'object' ? config : {})
    }
    typeCheckConfig(NAME, config, DefaultType)
    return config
  }
  
 
```
Make a check, first but I am pretty sure, `Manipulator.getDataAttributes` resolves automatically element 
`data-bs-root-element` to  `this._config.rootElement`

Reply:
Yep, that looks good! However `Manipulator.getDataAttributes()` will return `rootElement` as a string. I have kept the `getElement` method call to resolve the string selector to an element